runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectTimerInternals.java

```diff
@@ -45,11 +47,24 @@ private DirectTimerInternals(
   }
 
   @Override
+  public void setTimer(StateNamespace namespace, String timerId, Instant target,
+      TimeDomain timeDomain) {
+    // TODO: actually store by ID
```

Comment:
Probably should throw until timerId is supported, given that it's currently unused.